src/api/Gemfile.lock

```diff
@@ -152,7 +152,7 @@ GEM
       mime-types-data (~> 3.2015)
     mime-types-data (3.2016.0521)
     mini_portile2 (2.1.0)
-    minitest (5.9.0)
+    minitest (5.8.3)
```

Comment:
On Dienstag, 30. August 2016, 08:10:01 CEST wrote Mois�s D�niz Alem�n:
> > @@ -152,7 +152,7 @@ GEM
> >        mime-types-data (~> 3.2015)
> >      mime-types-data (3.2016.0521)
> >      mini_portile2 (2.1.0)
> > -    minitest (5.9.0)
> > +    minitest (5.8.3)
> 
> Then we should fix the version in the Gemfile

right, my fault .... can you fix that also in this commit?

-- 

Adrian Schroeter
email: adrian@suse.de

SUSE Linux GmbH, GF: Felix Imend�rffer, Jane Smithard, Graham Norton, HRB 21284 (AG N�rnberg) 
 
Maxfeldstra�e 5                         
90409 N�rnberg 
Germany